react/week2/todolist/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import logo from './logo.svg';
+import './App.css';
+
+let tasks = ["submit hyf homework", 
+            "go for a run", 
+            "watach movie"];
+
+            
+function TodoTimer(props){
+  const [count, setCount] = useState(0);
+  setTimeout(() => setCount(count + 1), 1000);
+  return <p>You have used this app for {count} seconds</p>;
+};
+
+function Todo(props){
+  
+  const [checked, updateCheck] = useState(false)
+  const taskDone = checked ? "done" : ""
+
+  return (
+    
+    <div className={taskDone}>
+      <input type="checkbox" onClick={() => {updateCheck(!checked)}}></input>
+      <label>{props.name}</label>
+    </div>
+  
+  )
+}
+
+function App() {
+
+  const [todos, updateTodos] = useState(tasks)
+
+
+  const onAdd = () => {
+    console.log ("adding new todo")
+    const newTodo  = document.getElementById("newTodo").value;
+    let newTodoList = todos.concat(newTodo)
+    updateTodos(newTodoList)
+    console.log(todos)
+  }
```

Comment:
```suggestion
  const onAdd = () => {
    const newTodo  = document.getElementById("newTodo").value;
    let newTodoList = todos.concat(newTodo)
    updateTodos(newTodoList)
  }
```
In this case you have access to the "tasks" array so you can pick one of those with a random ID rather than using document.getElementById("newTodo").

Reply:
ok. I changed it.